src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -411,10 +412,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('account_name', acct_name_type, id_part=None)
         c.argument('resource_group_name', required=False, validator=process_resource_group)
         c.argument('enable_change_feed', arg_type=get_three_state_flag(), min_api='2019-04-01')
+        c.argument('enable_container_delete_retention',
+                   arg_type=get_three_state_flag(),
+                   options_list=['--enable-container-delete-retention', '-c'],
```

Comment:
I personally feel that the semantics of `-c` is not clear enough
What about the alias name `-- container-delete`? This can be paired with `--container-days`

Reply:
Consider such options before, but for cx convenience, I choose `-c`. 
If we incline to have meaningful option, `--container-retention` should be better.

Reply:
OK